src/cli/domain/init-template/install-template.js

```diff
@@ -16,29 +15,27 @@ module.exports = function installTemplate(options, callback) {
     templateType
   } = options;
 
-  logger.log(strings.messages.cli.installCompiler(compiler));
+  const npmOptions = {
+    dependency: compiler,
+    installPath: componentPath,
+    isDev: true,
+    save: true
+  };
 
-  const args = ['install', '--save-dev', '--save-exact', compiler];
-  const installProc = spawn('npm', args, {
-    cwd: componentPath,
-    stdio: 'inherit'
-  });
+  logger.log(strings.messages.cli.installCompiler(compiler));
 
-  installProc.on('error', () => callback('template type not valid'));
-  installProc.on('close', code => {
-    if (code !== 0) {
-      return callback('template type not valid');
+  npm.installDependency(npmOptions, (err, result) => {
+    const errorMessage = 'template type not valid';
```

Comment:
Note: In the future we might want to improve those errors, I found that as we use the same error in different point make it quite hard to debug.